app/dubboctl/internal/builders/pack/build.go

```diff
@@ -67,7 +71,17 @@ var (
 		"ghcr.io/knative/",
 	}
 
-	defaultBuildpacks = map[string][]string{}
+	defaultBuildpacks = map[string][]string{
+		"go":   {"paketo-buildpacks/go"},
+		"java": {"paketo-buildpacks/java"},
+	}
+
+	// work around for resolving the images with latest tag could not be mirrored
+	latestImageTagMapping = map[string]string{
+		"paketobuildpacks/builder-jammy-tiny": "0.0.271",
+		"paketobuildpacks/run-jammy-tiny":     "0.2.46",
+		"buildpacksio/lifecycle":              "0.17.7",
```

Comment:
https://docker.aityp.com/ does not support `latest` tag, using tag mapping to resolve it.